packages/main/src/ProductSwitch.js

```diff
@@ -0,0 +1,102 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import ItemNavigation from "@ui5/webcomponents-base/dist/delegate/ItemNavigation.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import ProductSwitchTemplate from "./generated/templates/ProductSwitchTemplate.lit.js";
+
+// Styles
+import ProductSwitchCss from "./generated/themes/ProductSwitch.css.js";
+
+
+const metadata = {
+	tag: "ui5-productswitch",
+	properties: {
+		/**
+		 * Indicates how much columns should be displayed.
+		 * @private
+		 */
+		desktopColumns: {
```

Comment:
I don't like this API. My suggestion is that the columns count is defined depending on the size of the ProductSwitch itself